src/algorithms/rsa.rs

```diff
@@ -36,14 +37,12 @@ pub fn rsa_encrypt<K: PublicKeyParts>(key: &K, m: &BigUint) -> Result<BigUint> {
 pub fn rsa_decrypt<R: CryptoRngCore + ?Sized>(
     mut rng: Option<&mut R>,
     priv_key: &impl PrivateKeyPartsNew,
-    c_orig: &BigUint,
+    c: &BoxedUint,
```

Comment:
It might be worth leaving this as-is to preserve backwards compatibility, so we could potentially ship a v0.9.x patch release focused just on making `rsa_decrypt` constant-time (in theory)

Reply:
the public traits already have a breaking change, so I figured we might as well chnage the hazmat apis as well